Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Bindings/BasicConnectorBinding.cs

```diff
@@ -67,6 +67,8 @@ ITopLevelExceptionHandler topLevelExceptionHandler
 
   public void RemoveModel(ModelCard model) => _store.RemoveModel(model);
 
+  public void RemoveModels(List<ModelCard> models) => _store.RemoveModels(models);
```

Comment:
See corollary comment in FE PR: https://github.com/specklesystems/speckle-server/pull/4203#discussion_r1997193697

tl;dr: do we need this vs. removing models in a for loop?

Reply:
got my reply in the other pr